nats-base-client/jsclient.ts

```diff
@@ -398,7 +398,7 @@ export class JetStreamClientImpl extends BaseApiClient
           ErrorCode.ApiError,
         );
       }
-      jsi.config.deliver_subject = createInbox();
+      jsi.config.deliver_subject = createInbox(this.nc.options.inboxPrefix);
```

Comment:
so `this.nc.` does not have a create inbox function that makes use of the inbox Prefix automatically?
As is the case in go.

Reply:
no it doesn't - it is just a function. But anywhere it is called in the lib it is honored.